internal/cmd/kafka/command_topic_describe.go

```diff
@@ -55,9 +58,12 @@ func (c *authenticatedTopicCommand) describe(cmd *cobra.Command, args []string)
 		return err
 	}
 
-	numPartitions, err := c.getNumPartitions(topicName)
+	topicData, httpResp, err := kafkaREST.CloudClient.GetKafkaTopic(kafkaClusterConfig.ID, topicName)
```

Comment:
```suggestion
	topic, httpResp, err := kafkaREST.CloudClient.GetKafkaTopic(kafkaClusterConfig.ID, topicName)
```

Reply:
`kafkarestv3.TopicData` is its type too

Reply:
I'm ok either way. wdyt？

Reply:
I think it's easier to read this way. Up to you though!